### DISCUSSION TITLE
Input type Date

### DISCUSSION THREAD

**Discussion by LeSe0** [heroui-inc/heroui#676] [Help] [answered]
Hello! How can i customize opened date picker in input component ?

**Comment by GravyPouch** [comment]
You would have to do that manually with CSS

**Reply by LeSe0** [reply]
i cant find classname for date

**Comment by jrgarciadev** [comment, marked as answer]
Hey @LeSe0 due to the [Input](https://nextui.org/docs/components/input#input-types) `date` is the browser native picker you have to figure out how to customize it for each browser but it doesn't depend on `NextUI`, we hope to have the `Calendar/Date/Time` components in a near future based on the `react-aria` implementation  https://react-spectrum.adobe.com/react-aria/useDatePicker.html#styled-examples
